inc/saiswitch.h

```diff
@@ -331,6 +331,8 @@ typedef enum _sai_switch_failover_config_mode_t
  * per tunnel type global configuration.
  * SAI_OBJECT_TYPE_TUNNEL object configuration
  * overrides the switch scoped global configuration.
+ *
+ * @flags Contains flags
```

Comment:
this should not contain flags, please remove this, and make all attributes continuous